README.md

```diff
@@ -243,15 +243,25 @@ This shows a list of all slides on the left and a preview of the current slide o
 
 Key | Description
 ---|---
-Left Arrow | Go to previous slide
-Right Arrow | Go to next slide
+Left Arrow | Go to previous slide (Hide current step in [Appear](#appear) component)
+Right Arrow | Go to next slide (Show next step in [Appear](#appear) component)
 Space | Go to next slide
-Up Arrow | Hide current step in [Appear](#appear) component
-Down Arrow | Show next step in [Appear](#appear) component
 Option + P | Toggle [Presenter Mode](#presenter-mode)
 Option + O | Toggle [Overview Mode](#overview-mode)
 Option + G | Toggle grid view mode
 
+## Auto Transition
+
+To enable the automatic transitioning of your slides and appears, export a `autoTransition` constant.
+`autoTransition` should set the time between transitions.
+For example if you need to transition every 10 seconds, add the following to your MDX file
+
+```js
+// Example index.mdx
+
+export const autoTransition = 10
```

Comment:
Seems like a legitimate feature to add, but I'm not sure about it being configured in the mdx document. My thinking is that the configuration in mdx should be things that you *always* want set, whereas this feature seems more like something to enable in the app state – maybe via a button in presenter mode or a keyboard shortcut...

Reply:
Sure, will move this outside of the configuration.

Reply:
Yes, I like the idea of keeping the `slides.mdx` just for content & (basic-ish) layout, & deep configs external.